msp/msp_exp_handler.c

```diff
@@ -714,5 +729,8 @@ void Timer1_IRQHandler(void)
 	// citiroc_daq_set_citi_temp(hkadc_read(CITI_TEMP_CHAN));
 	citiroc_daq_set_hvps_volt(hvps_get_voltage());
 	citiroc_daq_set_hvps_curr(hvps_get_current());
-	MSS_TIM1_clear_irq();
+    MSS_TIM64_clear_irq();
+
+	/* Indicate TIM duration completion - blink LED with 300ms delay*/
+	led_blink(LED_BLINK_DAQ, 300);
```

Comment:
Not best-practice to add a delay inside an IRQ handler. Better to remove this call.

Reply:
I had put it there to only check for daq time. I am removing it. I do not think we require it there for the final code anyway.